From: "Chris \\\"Koying\\\" Browet" <cbro@semperpax.com>
Date: Mon, 21 Mar 2011 08:54:23 +0100
Subject: FIX : HSL stuff not defined in Qt < 4.6

---
 plugins/background/MGdalBackground/GdalAdapter.cpp |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/plugins/background/MGdalBackground/GdalAdapter.cpp b/plugins/background/MGdalBackground/GdalAdapter.cpp
index af59374..9888d04 100644
--- a/plugins/background/MGdalBackground/GdalAdapter.cpp
+++ b/plugins/background/MGdalBackground/GdalAdapter.cpp
@@ -361,6 +361,7 @@ bool GdalAdapter::loadImage(const QString& fn)
                 theImg.setPixel(px, py, qRgba(*r, *g, *b, a));
                 break;
             }
+#if QT_VERSION >= 0x040600
             case GdalAdapter::Hsl:
             {
                 float* h = lineBuf + (col*bandCount) + ixH;
@@ -375,6 +376,7 @@ bool GdalAdapter::loadImage(const QString& fn)
                 theImg.setPixel(px, py, C.rgba());
                 break;
             }
+#endif
             case GdalAdapter::Cmyk:
             {
                 float* c = lineBuf + (col*bandCount) + ixC;
@@ -422,6 +424,7 @@ bool GdalAdapter::loadImage(const QString& fn)
                 theImg.setPixel(px, py, qRgba(color->c1, color->c2, color->c3, color->c4));
                 break;
             }
+#if QT_VERSION >= 0x040600
             case GdalAdapter::Palette_HLS:
             {
                 float* ix = (lineBuf + (col*bandCount));
@@ -430,6 +433,7 @@ bool GdalAdapter::loadImage(const QString& fn)
                 theImg.setPixel(px, py, C.rgba());
                 break;
             }
+#endif
             case GdalAdapter::Palette_CMYK:
             {
                 float* ix = (lineBuf + (col*bandCount));
-- 
